ert_gui/plottery/plot_config.py

```diff
@@ -7,11 +7,12 @@ class PlotConfig(object):
     # The plot_settings input argument is an internalisation of the (quite few) plot
     # policy settings which can be set in the configuration file.
     
-    def __init__(self, plot_settings, title="Unnamed", x_label=None, y_label=None):
+    def __init__(self, plot_settings=None, title="Unnamed", x_label=None, y_label=None):
```

Comment:
If plot_settings are not used anymore, could it not be removed from the parameter-list?

Reply:
I left it there in order it can be set from code if required.